packages/react/src/components/date-picker/utils/datepicker-utils.ts

```diff
@@ -65,3 +65,18 @@ export function setLocaleFirstDayOfWeek(locale: Locale, dayOfWeek?: DayOfWeek):
     const optionsOverride: Locale['options'] = { weekStartsOn: dayOfWeek };
     Object.assign(locale.options || {}, optionsOverride);
 }
+
+/**
+ * The version of format as only numbers (ex: "yyyy-MM-dd" would become "yyyyMMdd").
+ * Note that Month and Day are always converted to the 2 digits format.
+ */
+export function getNumericalDateFormat(dateFormat: string): string | void {
+    if (/[^yMd/\-,. ]/g.test(dateFormat)) {
+        return;
+    }
```

Comment:
Je pense que ce serait mieux que la fonction retourne un nullable plutôt que void
```suggestion
export function getNumericalDateFormat(dateFormat: string): string | null {
    if (/[^yMd/\-,. ]/g.test(dateFormat)) {
        return null;
    }
```

Reply:
Bien vu, on s'attend a un resultat.